src/test/java/guitests/FindCommandTest.java

```diff
@@ -18,6 +18,15 @@ public void find_nonEmptyList() {
         commandBox.runCommand("delete 1");
         assertFindResult("find Meier",td.daniel);
     }
+    
+    @Test
+    public void find_nonEmptyList_byTag() {
+        assertFindResult("find friends", td.alice, td.benson); //multiple results
+
+        //find after deleting one result
+        commandBox.runCommand("delete 1");
+        assertFindResult("find friends", td.benson);
+    }
 
     @Test
     public void find_emptyList(){
```

Comment:
There should be a whitespace between 'emptyList()' and '{'
`public void final_emptyList() {`